test/cypress/integration/user/transaction_details_spec.js

```diff
@@ -141,6 +143,19 @@ describe('Transactions details page', () => {
       cy.get('.transaction-details tbody').find('tr').eq(14).find('td').eq(1).should('have.text',
         chargeWithDelayedCapture.email)
     })
+
+    it('should display corporate card surcharge in the amount field correctly when there is a corporate card surcharge', () => {
+      cy.visit(`${transactionsUrl}/${aCorporateCardSurchargeCharge.charge_id}`)
+
+      // Ensure page title is correct
+      cy.title().should('eq', `Transaction details ${aCorporateCardSurchargeCharge.reference} - System Generated test - GOV.UK Pay`)
+
+      // Ensure page details match up
+
+      // Amount
+      cy.get('.transaction-details tbody').find('tr').eq(3).find('td').eq(1).should('have.text',
```

Comment:
Wouldn’t this work?

```suggestion
      cy.get('#amount').should('have.text',
```

Reply:
This new suggestion feature is neat, you can just apply my update with a commit within Github if you think it’s legit